### PR TITLE
[Variable Fonts article] Add oldschool hacks tip

### PR DESCRIPTION
**Pull request by tomayac**
Changes proposed in this pull request:

- Add oldschool hacks tip (verified by native speaker).

### CODE REVIEW

**Review by rachelandrew — APPROVED** [review]

File: src/site/content/es/blog/variable-fonts/index.md [hunk 1]
```diff
@@ -450,6 +450,8 @@ body {
 
 Para los navegadores más antiguos, el texto con la clase `.super-bold` se renderizará en negrita normal, ya que esa es la única fuente en negrita que tenemos disponible. Cuando se tiene compatibilidad con fuentes variables, podemos usar 1000, el cual es el mayor peso posible.
```
[the remark this answers is not in the document]
Comment (wordsmeet) [reply src/site/content/es/blog/variable-fonts/index.md:451]: Sorry @tomayac, I don't understand what you're trying to say here.

[the remark this answers is not in the document]
Comment (tomayac) [reply src/site/content/es/blog/variable-fonts/index.md:451]: Just that this suggestion is part of a new pull request.

[the remark this answers is not in the document]
Comment (wordsmeet) [reply src/site/content/es/blog/variable-fonts/index.md:451]: Thank you, I'm still unfamiliar with how the whole process goes. If there are other Spanish paragraphs in need of a translator's revision, please let me know.

[the remark this answers is not in the document]
Comment (tomayac) [reply src/site/content/es/blog/variable-fonts/index.md:451]: There’s no formal process. If you apot something, you can click “Improve article” at the bottom, make your edits, and suggest them to be merged by opening a pull request.

**Pull request merged**